clippy_lints/src/utils/mod.rs

```diff
@@ -1060,5 +1060,7 @@ pub fn get_arg_name(pat: &Pat) -> Option<ast::Name> {
 
 /// Returns alignment for a type, or None if alignment is undefined
 pub fn alignment<'a, 'tcx>(cx: &LateContext<'a, 'tcx>, ty: Ty<'tcx>) -> Option<Align> {
-    (cx.tcx, cx.param_env).layout_of(ty).ok().map(|layout| layout.align)
+    cx.tcx.layout_of(cx.param_env.and(ty))
```

Comment:
You don't need to do this, here or above. You can use `cx.layout_of(ty)` directly. And I'd suggest inlining these functions instead of having them abstracted here.

Reply:
done 503a63390df10c6c025f1c1c514a232a0a163c38